src/Bookings.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "./data/fakeBookings.json";
```

Comment:
So this line is no longer needed as you imported `FakeBookings` into another file where you referenced it - this was `SearchResults.js`. If you look at the top of your file then you'll see that it is slightly faded out - this is VSCode telling you that you've declared a variable in your code which you're not using.